src/test/java/org/reaktivity/specification/http/rfc7230/ConnectionManagementIT.java

```diff
@@ -274,6 +274,18 @@ public void concurrentUpgradeRequestsandResponsesWithData() throws Exception
 
     @Test
     @Specification({
+        "${scripts}/request.incomplete.response.headers.and.abort/client",
+        "${scripts}/request.incomplete.response.headers.and.abort/server" })
+    @ScriptProperty("serverTransport \"nukleus://http/streams/source\"")
+    public void requestAndAbortBeforeResponseHeadersComplete() throws Exception
```

Comment:
Some method names like clientAndServerMustCloseConnectionAfterRequestWithConnectionClose or serverMustCloseConnectionAfterResponseWithConnectionClose make use of the word "must" to indicate the nature of the requirement in the HTTP specification. Are you okay with names like "mustXxx" in addition to "shouldXxx"? Alternative is to lose the "must" information if you think it is unimportant. Also, what do you suggest for names like serverMustCloseConnectionAfterResponseWithConnectionClose? "shouldCloseConnectionAfterResponseWithConnectionClose" loses some meaning. "shouldCloseConnectionAfterResponseWithConnectionCloseWhenActingAsServer" as little bit clumsy but I could live with it. Can you?